mailhandler/spameater

```diff
@@ -413,6 +418,7 @@ $config->debug('actually not... the account is disabled.');
     $st->bind_columns(\%attr,\$UserID,\$RealEmail,\$SpamEmail,\$Prefix,\$Features,
      \$RecThrottleTime,\$RecThrottleCount,\$DefaultNumber);
     $st->fetch();
+    $config->debug("found real email to deliver to $RealEmail with userid $UserID") if $extradebug;
```

Comment:
I know these are debug logs, but I don't know where this logs will be stored or how they might be accessed. To prevent the potential of leaking PII (Personally Identifying Information), it might be better to obfuscate the real email in the logs.

Reply:
Think there's a bigger conversation about that and that we likely have to turn on monitoring in order to be able to troubleshoot which mail providers aren't getting good enough service. When we do that we have to resolve all PII in troubleshooting logs and statistics. 

In the meantime this is a setting which must be turned on *only* in testing, not in production.

Reply:
In other words: _please open a ticket about monitoring and PII_